code/datums/mob_stats.dm

```diff
@@ -43,7 +43,8 @@
 /datum/stat_holder/proc/getStat(statName, pure = FALSE)
 	if (!islist(statName))
 		var/datum/stat/S = stat_list[statName]
-		SEND_SIGNAL(holder, COMSIG_STAT, S.name, S.getValue(), S.getValue(TRUE))
+		if(holder)
```

Comment:
Holder should never be null, diagnose this further.

Reply:
I've already tried and found nothing except that mob stats are never destroyed.